### PR TITLE
Created Journey Book readme

### PR DESCRIPTION
**Pull request by aryav-v**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: journeys/Aryav-Chaturvedi.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h1><a href="https://github.com/aryav-v">Aryav Chaturvedi's</a> Journey</h1>
```
Comment (KendallDoesCoding) [inline journeys/Aryav-Chaturvedi.md:1]:
```suggestion
# [Aryav Chaturvedi](https://github.com/aryav-v)'s Journey
```

**Review by KendallDoesCoding — COMMENTED** [review]
dont use html in md

**Comment by aryav-v** [comment]
Is the issue resolved , or should I edit the readme again?

**Comment by KendallDoesCoding** [comment]
> Is the issue resolved , or should I edit the readme again?

The issue is resolved. Thanks

**Pull request merged**